SPECS/libnbd/libnbd.spec

```diff
@@ -122,26 +170,51 @@ Install this package if you want intelligent bash tab-completion
 for %{name}.
 
 %prep
+%if 0%{verify_tarball_signature} && ! 0%{?azl}
+%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%endif
 %autosetup -p1
+%if 0%{patches_touch_autotools} && ! 0%{?azl}
+autoreconf -i
+%endif
+
 
 %build
 %configure \
     --disable-static \
     --with-tls-priority=@LIBNBD,SYSTEM \
+    --with-bash-completions \
     PYTHON=%{__python3} \
     --enable-python \
     --enable-ocaml \
     --enable-fuse \
-    --disable-golang
+    --disable-golang \
+    --disable-rust \
+    %{!?azl:--enable-ublk}
+
 %make_build
 
+
 %install
 %make_install
-find %{buildroot} -type f -name '*.la' -print -delete
+
+# Delete libtool crap.
+find %{buildroot} -name '*.la' -delete
 
 # Delete the golang man page since we're not distributing the bindings.
 rm %{buildroot}%{_mandir}/man3/libnbd-golang.3*
 
+%ifarch %{ix86}
+# Delete the OCaml man page on i686.
+rm %{buildroot}%{_mandir}/man3/libnbd-ocaml.3*
+%endif
+
+%if 0%{?rhel} || 0%{?azl}
+# Delete nbdublk on RHEL.
```

Comment:
Comment seem incorrect, since we delete it for Azure Linux as well.

Reply:
```suggestion
# Delete nbdublk on RHEL and azl
```